### PR TITLE
Add simpleTag to Redux DefaultState interface

### PR DESCRIPTION
**Pull request by adrw**
* Migrate Redux related code from `@misk/common`

### CODE REVIEW

File: misk-web/web/packages/@misk/core/src/ducks/index.ts [hunk 1]
```diff
@@ -1,8 +1,28 @@
+import { fromJS, List } from "immutable"
 export * from "./simpleDucksUtilities"
 export * from "./routerDucks"
 export * from "./simpleFormDucks"
 export * from "./simpleNetworkDucks"
 
+// from @misk/common/src/index.ts
```
Comment (wesleyk) [inline misk-web/web/packages/@misk/core/src/ducks/index.ts]: what does this mean?

File: misk-web/web/packages/@misk/core/src/ducks/index.ts [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+export interface IDefaultState {
+  data: any
+  error: any
+  loading: boolean
+  simpleTag: string
+  success: boolean
+  toJS?: () => any
+}
+
+export const createDefaultState = (simpleTag: string): IDefaultState =>
```
Comment (wesleyk) [inline misk-web/web/packages/@misk/core/src/ducks/index.ts]: can we add a comment documenting this if it's part of our API?

File: misk-web/web/packages/@misk/core/src/ducks/index.ts (hunk 1 above)
Comment (adrw) [reply misk-web/web/packages/@misk/core/src/ducks/index.ts]: This code used to live in `@misk/common`

File: misk-web/web/packages/@misk/core/src/ducks/index.ts (hunk 2 above)
Comment (adrw) [reply misk-web/web/packages/@misk/core/src/ducks/index.ts]: Yes will do

File: misk-web/web/packages/@misk/core/src/ducks/index.ts (hunk 1 above)
Comment (wesleyk) [reply misk-web/web/packages/@misk/core/src/ducks/index.ts]: and it no longer lives there? I think this comment adds more confusion than clarity.

File: misk-web/web/packages/@misk/core/src/ducks/index.ts [hunk 3]
```diff
@@ -1,9 +1,61 @@
+import { fromJS, List } from "immutable"
 export * from "./simpleDucksUtilities"
 export * from "./routerDucks"
 export * from "./simpleFormDucks"
 export * from "./simpleNetworkDucks"
 
-// from @misk/common/src/actions.ts
+/**
```
Comment (wesleyk) [inline misk-web/web/packages/@misk/core/src/ducks/index.ts]: so what is the API being exposed by this file? Should developers be familiar with the entire file and each type exposed by it? Goes back to the metapoint about making a concise API surface area so we don't overwhelm developers with code to understand

Comment (adrw) [reply misk-web/web/packages/@misk/core/src/ducks/index.ts]: Fair point, will address this in a future PR as we formalize the API contracts for other parts of the framework. Will track progress here #177

**Pull request merged**